### PR TITLE
Functional tests - Add test 'Bulk update orders status'

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Add functional test 'Bulk update orders status'
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | `TEST_PATH="functional/BO/02_orders/01_orders/06_bulkUpdateOrdersStatus.js" URL_FO=shopUrl/ npm run specific-test`

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/18350)

### CODE REVIEW

File: tests/puppeteer/campaigns/functional/BO/02_orders/01_orders/06_bulkUpdateOrdersStatus.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        [1, 2],
+      );
+      await expect(textResult).to.equal(this.pageObjects.ordersPage.successfulUpdateMessage);
+    });
+
+    ['first', 'second'].forEach((arg, index) => {
+      it(`should check ${arg} order status`, async function () {
+        await testContext.addContextItem(this, 'testIdentifier', `checkOrderStatus${index + 1}`, baseContext);
+        const orderStatus = await this.pageObjects.ordersPage.getTextColumn('osname', index + 1);
+        await expect(orderStatus, 'Order status is not correct').to.equal(Statuses.paymentAccepted.status);
```
Comment (Progi1984) [inline tests/puppeteer/campaigns/functional/BO/02_orders/01_orders/06_bulkUpdateOrdersStatus.js:144]: The label is not correct. Isn't it ?

Comment (boubkerbribri) [reply tests/puppeteer/campaigns/functional/BO/02_orders/01_orders/06_bulkUpdateOrdersStatus.js:144]: `Order status is not correct` is the error message that will display if the expect fails (if that what you are talking about).

Comment (NeOMakinG) [reply tests/puppeteer/campaigns/functional/BO/02_orders/01_orders/06_bulkUpdateOrdersStatus.js:144]: Do we have some more informations if it fails ? like " Expected waiting for payment, got waiting for delivery "

Comment (boubkerbribri) [reply tests/puppeteer/campaigns/functional/BO/02_orders/01_orders/06_bulkUpdateOrdersStatus.js:144]: yes, when fail, we have expected value and actual one, like this 
image

Comment (NeOMakinG) [reply tests/puppeteer/campaigns/functional/BO/02_orders/01_orders/06_bulkUpdateOrdersStatus.js:144]: That should be fine then

**Review by NeOMakinG — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Pull request merged**